inventory/sample/hosts.ini

```diff
@@ -0,0 +1,11 @@
+[host_vm]
+rpi-creation    ansible_host=10.182.31.179   ansible_port=22     ansible_user=ubuntu
+
+[master]
+ubuntu          ansible_host=10.182.31.179    ansible_port=5555   ansible_user=rpi_admin rpi_static_ip=10.0.0.10
```

Comment:
This should ideally be dynamically generated in an Ansible task.
But for now, could you provide the default values to be:
```
qemu          ansible_host=10.182.31.179  ansible_user=admin  ansible_port=5555    rpi_static_ip=10.0.0.10
```
or is the hostname of importance and must be `ubuntu`?

Reply:
@chinzou ☝️ ?
> is the hostname of importance and must be ubuntu?